monitor/dbt_runner.py

```diff
@@ -51,12 +51,21 @@ def snapshot(self) -> bool:
         success, _ = self._run_command(['snapshot'])
         return success
 
-    def run_operation(self, macro_name: str, json_logs: bool = True, macro_args: dict = None,
-                      log_errors: bool = False) -> list:
+    def run_operation(
+        self,
+        macro_name: str,
+        json_logs: bool = True,
+        macro_args: Optional[dict] = None,
+        log_errors: bool = False,
+        dbt_vars: Optional[dict] = None
+    ) -> list:
         command_args = ['run-operation', macro_name]
         if macro_args is not None:
             json_args = json.dumps(macro_args)
             command_args.extend(['--args', json_args])
+        if dbt_vars is not None:
```

Comment:
Since vars is supported by all dbt commands, I'd implement this code inside _run_command to avoid the code duplication